src/pages/task/logs/logMessageLine/LogLines.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from "@emotion/styled/macro";
+
+export const DebugMessage = styled.div`
+  color: #666;
+`;
+
+export const InfoMessage = styled.div`
+  color: #333;
+`;
+
+export const WarnMessage = styled.div`
+  color: #ffa500;
+`;
+
+export const ErrorMessage = styled.div`
+  color: #ff0000;
+`;
+
+export const DMessage = styled.div`
+  color: #666;
+`;
+
+export const IMessage = styled.div`
+  color: #333;
+`;
+
+export const WMessage = styled.div`
+  color: #ffa500;
+`;
+
+export const EMessage = styled.div`
+  color: #ff0000;
+`;
```

Comment:
oh wait there are much more than four hahaa. def think one component with a color prop would be wise.